src/Lean/Server/FileWorker/WidgetRequests.lean

```diff
@@ -17,6 +17,15 @@ import Lean.Server.FileWorker.RequestHandling
 namespace Lean.Widget
 open Server
 
+builtin_initialize
+  registerBuiltinRpcProcedure
+    `Lean.Widget.ppExprTagged
+    (WithRpcRef ExprWithCtx × Bool)
```

Comment:
Please use a structure for future extensibility.

Reply:
@Vtec234 please could you check if I've put the WithRpcRef in the right place?

Reply:
Yep! Maybe `PPExprTaggedParams` to follow the LSP conventions (not that I entirely followed them so far, but it's never too late!).